spec/apps/partial-rerouting/partial-rerouting.spec.js

```diff
@@ -2,7 +2,7 @@ import * as singleSpa from "single-spa";
 
 describe("partial rerouting", () => {
   function delay() {
-    return new Promise((resolve) => setImmediate(resolve));
```

Comment:
setImmediate has issues in jest 27. See https://github.com/facebook/jest/issues/11539